File: v-lang/lang.java/Spring/v-mybatis/src/main/java/org/example/controller/FooController.java
package org.example.controller;

import jakarta.annotation.Resource;
import org.example.mapper.FooMapper;
import org.example.model.Foo;
import org.springframework.web.bind.annotation.PostMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class FooController {

    @Resource
    private FooMapper fooMapper;

    @PostMapping("/test")
    public void insert() {
        Foo foo = Foo.builder()
                .name("lkzc19")
                .build();
        fooMapper.insert(foo);
    }
}

File: v-lang/l-kotlin/v-spring/spring-web/src/main/kotlin/org/example/controller/FooController.kt
package org.example.controller

import org.example.controller.param.FooParam
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.PostMapping
import org.springframework.web.bind.annotation.RequestBody
import org.springframework.web.bind.annotation.RequestParam
import org.springframework.web.bind.annotation.RestController

@RestController
class FooController {

    @GetMapping("/foo.get")
    fun foo(@RequestParam name: String) = "foo...${name}"

    @PostMapping("/foo.post")
    fun foo(@RequestBody param: FooParam) = "foo...${param.name}"
}